pkg/util/util.go

```diff
@@ -35,7 +37,16 @@ func GetEncryptionTransformers(encryptionConfigSecretName string, secrets v1core
 	if !ok {
 		return transformerMap, fmt.Errorf("no encryptionConfig provided")
 	}
-	return encryptionconfig.ParseEncryptionConfiguration(bytes.NewReader(encryptionConfigBytes))
+	err = ioutil.WriteFile("encryptionConfig.yaml", encryptionConfigBytes, os.ModePerm)
```

Comment:
Nit: declare the filename as a const at the top of the file instead of a hardcoding value in the code.